docs/AdviceForOssBeginners.md

```diff
@@ -0,0 +1,26 @@
+# Advice for OSS Beginners
+
+//TODO: flesh out these points
```

Comment:
Is this TODO still valid? If only two points are not fleshed out, the TODO should be moved there instead.
If necessary I can come up with some good descriptions for those points.